### ISSUE TITLE
MS4.4.0 Doesn't work anymore

### ISSUE THREAD

**Issue by tharosd** [XiaoMigros/Tap-BPM#1]
Since 4.4.0 update, it seems the plugin doesn't work anymore. No dialog window opens

**Comment by XiaoMigros** [comment]
Yes, sorry about that! All plugins need to be adapted to work for 4.4 and I haven't got around to doing that just yet :)

**Comment by tharosd** [comment]
No problem. It's just that with the previous version 4.3.2 everything worked fine. Since this afternoon's update to 4.4.0, the plugin no longer opens.

**Comment by XiaoMigros** [comment]
I'm well aware and will get to a fix over the coming days. Thanks for logging!
